app/models/base_model.rb

```diff
@@ -14,4 +14,8 @@ def update_file!(name)
     raw_write_attribute(:filename, name)
     save!
   end
+
+  def public_stories(arel = nil)
```

Comment:
Ah, gotcha - was forgetting that we often access story-counts from the other side of the relation.  (`series.public_stories.count` ... instead of `Story.public_stories.where(series: series).count`).  Yeah, I think having it here is fine.